packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -851,6 +851,139 @@ void main() {
     expect(tester.getTopLeft(find.text('Item 0')).dx, 72);
     expect(tester.getTopLeft(find.text('Item 1')).dx, 72);
   });
+
+  test('PopupMenuButton takes only child or only icon', () {
```

Comment:
This is incorrectly described, since you can also have both be undefined and it would use the default icon.
```suggestion
  test('PopupMenuButton's child and icon properties cannot be simultaneously defined', () {
```